src/main/java/Mediell/Task.java

```diff
@@ -0,0 +1,61 @@
+package Mediell;
+
+import java.util.Objects;
+
+/** Represents a Task. */
+public class Task {
+    private String taskName;
+    private boolean completed;
```

Comment:
Should completed be renamed to isComplete to following naming conventions for booleans?